walkthroughs/howto-k8s-ingress-gateway/README.md

```diff
@@ -0,0 +1,118 @@
+# Configuring Ingress Gateway
+
+In this walkthrough, we'll configure an Ingress Gateway in our existing example color app but with a VirtualGateway resource instead of VirtualNode for ingress traffic.
+
+A virtual gateway allows resources outside your mesh to communicate to resources that are inside your mesh. The virtual gateway represents an Envoy proxy running in an Amazon ECS, in a Kubernetes service, or on an Amazon EC2 instance. Unlike a virtual node, which represents a proxy running with an application, a virtual gateway represents the proxy deployed by itself.
+
+## Prerequisites
+This feature is currently only available in App Mesh preview and will work with App Mesh manager [here](https://github.com/aws/eks-charts/tree/master/stable/appmesh-manager)
```

Comment:
Note: We still need to produce a managed version of the controller image which works with preview (the same image cannot be used for both, since it's set at compile time). See aws/aws-app-mesh-controller-for-k8s#75 for that remaining work.

Reply:
We will publish release candidates for appmesh and appmesh-preview. Kept this PR in draft until the release candidates are published

Reply:
Did we publish these? I see the PR is merged, but the instructions were not updated with how to get the preview version specifically.

Reply:
@bcelenza and I chatted about this offline. Adding a comment for everyone else. The current default is set to preview [1] and the README [2] has a note on what needs to be changed to use the preview. It's a bit confusing as the charts are not published so the only way to install is using local build. We will be publishing the charts soon

[1] https://github.com/aws/eks-charts/blob/master/stable/appmesh-manager/values.yaml#L10
[2] https://github.com/aws/eks-charts/blob/master/stable/appmesh-manager/README.md#prerequisites